Api/Services/EmploymentService.cs

```diff
@@ -44,4 +52,48 @@ public async Task<Result<bool>> DeleteEmploymentAsync(int employmentId, string u
 
         return true;
     }
+    /// <summary>
+    /// Terminates all employments specified through ids in the list
+    /// </summary>
+    /// <param name="employmentIds"></param>
+    /// <param name="user"></param>
+    /// <returns></returns>
+    public async Task<Result<bool>> DeleteBulkEmploymentAsync(List<int> employmentIds, User user)
+    {
+
+        var employments = new List<Employment>();
+        foreach (var employmentId in employmentIds)
+        {
+            var employment = await context.Employments
+            .Include(e => e.Restaurant)
+            .ThenInclude(r => r.Group)
+            .FirstOrDefaultAsync(e => e.Id == employmentId && e.DateUntil == null);
```

Comment:
Z takim warunkiem będziemy dostawać NotFound jeśli Employment już jest zakończony. Lepiej jest zrobić oddzielnego ifa gdzie jeśli `DateUntil != null`, to przepuskamy ten Employment

Reply:
Czy celem tej końcówki nie jest zwrócenie tylko aktywnych zatrudnionych?